inference-engine/src/mkldnn_plugin/ngraph_transformations/convert_to_cpu_specific_opset.hpp

```diff
@@ -15,12 +15,21 @@
 #include "convert_to_swish_cpu.hpp"
 #include "reshape_prelu.hpp"
 #include "rnn_sequences_optimization.hpp"
+#include "move_eltwise_up_data_movement.hpp"
 
 namespace MKLDNNPlugin {
 
 inline void ConvertToCPUSpecificOpset(std::shared_ptr<ngraph::Function> &nGraphFunc) {
     ngraph::pass::Manager manager;
     manager.register_pass<ngraph::pass::ConstantFolding>();
+    manager.register_pass<MoveEltwiseUpThroughDataMov>();
+
+    manager.get_pass_config()->set_callback<MoveEltwiseUpThroughDataMov>([](const std::shared_ptr<const ngraph::Node>& node) -> bool {
+        if (node->get_input_size() >= 2) {
+            return node->get_input_element_type(1) != ngraph::element::i8 && node->get_input_element_type(1) != ngraph::element::u8;
+        }
+        return true;
+    });
```

Comment:
Could you please tell why do we place it in `ConvertToCPUSpecificOpset()` it seems that this transformation do not use CPU specific opset. We have `Transformation()` function in mkldnn_plugin.cpp looks like a more suitable place.